azure/elastic-datastore-sample/pom.xml

```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>com.yanchware</groupId>
             <artifactId>fractal.sdk</artifactId>
-            <version>1.2.2</version>
+            <version>1.5.4</version>
```

Comment:
Could it make sense to use parent POM?
To set this version only once?

Reply:
As discussed, let's analyze it in a future task